src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -725,26 +725,28 @@ public void onSuccess(Boolean result) {
    */
   private ListenableFuture<Boolean> initializeDraft() {
 
-    final SettableFuture<Boolean> result = new SettableFuture<>();
+    SettableFuture<Boolean> result = new SettableFuture<>();
 
-    final String    draftText      = getIntent().getStringExtra(TEXT_EXTRA);
-    final Uri       draftMedia     = getIntent().getData();
```

Comment:
getIntent().getData() gets set here https://github.com/deltachat/deltachat-android/blob/d10891311fa706827f3ec50263c1298a7537332b/src/org/thoughtcrime/securesms/NewConversationActivity.java#L171, if a sendto link was clicked. 
Replacing it by `RelayUtil.getSharedUris(this);` doesn't work in that case, because the latter will return an empty list.